docs/elements.md

```diff
@@ -642,6 +644,63 @@ Attribute | Type | Default | Description
 
 -----
 
+### `pl-matching` element
+
+Given a list of statements, select a matching option for each entry from a drop-down list.
+
+#### Sample element
+
+![](elements/pl-matching.png)
+
+**question.html**
+```html
+<pl-matching answers-name="string_value">
+  <pl-statement match="d.c.">United States</pl-statement>
+  <pl-statement match="Mexico City">Mexico</pl-statement>
+  <pl-statement match="Paris">France</pl-statement>
+
+  <pl-option name="d.c.">Washington, D.C.</pl-option>
+  <pl-option>Mexico City</pl-option>
+  <pl-option>Paris</pl-option>
+</pl-matching>
+```
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`answers-name` | string | — | Variable name to store data in.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
+`fixed-order` | boolean | False | Whether or not to display the statements in a fixed order; otherwise they are shuffled. Options are always shuffled.
+`number-statements` | integer | special | The number of statements to display. Defaults to all statements.
+`number-options` | integer | special | The number of options to display. Defaults to all options.
+`none-of-the-above` | boolean  | false | Whether or not to add a "None of the above" to the end of the options.
+`blank` | boolean | False | Option to add blank dropdown entry as the default selection in each drop-down list..
```

Comment:
```suggestion
`blank` | boolean | False | Option to add blank dropdown entry as the default selection in each drop-down list.
```

I'm wondering also if it should default to True instead, which is the default behaviour of `pl-dropdown`. What do you think?

Reply:
I'm fine with that.